### PR TITLE
Minor changes to User Guide

### PR DESCRIPTION
**Pull request by nweiyue**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -315,7 +315,7 @@ bye
 ### <a name="ucp"></a>User confirmation prompt
 
 Prompts the user to confirm the execution of commands that may permanently remove information.
-These commands are: `deletestu`, `editses` and `clearstu`.
+These commands are: `deletestu`, `editstu`,`clearstu`, `deleteses`, `editses`,
```
Comment (erisjacey) [inline docs/UserGuide.md]: Perhaps the last comma could be changed to a full stop?

**Review by erisjacey — COMMENTED** [review]
LGTM! Just one small nitpick.

File: docs/UserGuide.md (hunk 1 above)
Comment (nweiyue) [reply docs/UserGuide.md]: Good find! Will merge after resolving!

**Review by marcustw — APPROVED** [review]
LGTM!

**Pull request merged**